### PR TITLE
Home page enhancements

### PR DESCRIPTION
**Pull request by mkewls**
** Animated background for landing page
** Added copy for about section
** Some image and style changes in the about section

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "email-templates": "^2.5.4",
     "express": "^4.14.0",
     "express-session": "^1.14.2",
+    "html-loader": "^0.4.4",
```
Comment (jakepeyser) [inline package.json]: loaders should all be dev dependencies

File: app/components/home/Home.js [hunk 2]
```diff
@@ -128,11 +133,21 @@ export default () => (
...
+            <img
+              src={require('../../../src/images/sunsetjump.png')}
+              className="img-circle img-responsive"
+              style={ aboutImagesStyle }/>
           </div>
           <div className="content-text">
-            <h3>Our Methodology</h3>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sit amet erat at neque accumsan malesuada in et purus. Suspendisse potenti. Aenean et tellus a turpis gravida sodales id a enim. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Praesent tortor velit, accumsan sit amet maximus iaculis, condimentum quis magna. Fusce ac nulla tempor, vehicula risus vel, feugiat elit. Nulla ac quam ligula. Praesent pellentesque et sapien non fringilla.</p>
+          <h1>Our Vision</h1>
+          <p>We're a team that likes to solve problems. Our decision to build a digital product focused on
```
Comment (jakepeyser) [inline app/components/home/Home.js]: Good copy, but maybe just a few sentences long for the home page IMHO

File: app/components/home/Home.js [hunk 3]
```diff
@@ -128,11 +133,21 @@ export default () => (
       <div className="section">
         <div className="section-content">
           <div className="img-wrapper">
-            <img src={require('../../../src/images/work.jpg')} id="work"></img>
+            <img
+              src={require('../../../src/images/sunsetjump.png')}
```
Comment (jakepeyser) [inline app/components/home/Home.js]: I really like this until it gets to mobile. Not sure what we'd want to replace it with then

File: package.json (hunk 1 above)
Comment (jakepeyser) [inline package.json]: Actually do we need any of these anymore since we're just using a gif?

Comment (mkewls) [inline package.json]: We don't. Good catch. Never uninstalled from my testing of various approaches.

File: app/components/home/Home.js (hunk 2 above)
Comment (mkewls) [inline app/components/home/Home.js]: Haha. Don't micromanage my copy! Will try to tighten it up.

File: app/components/home/Home.js (hunk 3 above)
Comment (jakepeyser) [inline app/components/home/Home.js]: I like an image on mobile. I find the gif strange on mobile b/c I can only see the woman's hands/feet clipping the frame every so often

**Pull request merged**